integration_test/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+TESTDIR=/tmp/rust_miniscript_test
```

Comment:
In dd322eb8fb9d7732b4e6ec7ca8c0213763ba4200:

This is fine for now but we should use a real single-use temp name. This is going to conflict e.g. whenever you and I are simultaneously running these tests on my big server.